mex/model/vocabularies/bibliographic-resource-type.json

```
@@ -0,0 +1,265 @@
+[
+    {
+        "altLabel": {
+            "de": "Monografie",
+            "en": "Monograph"
+        },
+        "definition": {
+            "de": "Eine Monographie.",
+            "en": "A monograph."
+        },
+        "exactMatch": [
+            "https://schema.org/Book",
+            "http://www.wikidata.org/entity/Q571",
+            "http://purl.org/ontology/bibo/Book"
+        ],
+        "identifier": "https://mex.rki.de/item/bibliographic-resource-type-1",
+        "inScheme": "https://mex.rki.de/item/bibliographic-resource-type",
+        "prefLabel": {
+            "de": "Buch",
+            "en": "Book"
+        }
+    },
+    {
+        "altLabel": {
+            "en": [
+                "Book section"
+            ]
```

```suggestion
            "en": "Book section"
```

this can't be a list

---

also applies to the altLabels below. (maybe we should also have a schema for the vocabs 🤔)